### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by IvanCherepov**
Fix multiple typos in the documentation.

### CODE REVIEW

**Review by michaelmdresser — CHANGES_REQUESTED** [review]
Thank you so much for this PR! Documentation quality is really important to us and it is a continuing battle to keep it all up-to-date and well-written. All the help we can get is appreciated. Do you have any recommendations for linting documentation PRs?

Three things about this PR:
- Our current primary branch for this repo is actually `main`, but we haven't yet made it the default in GitHub. If its possible to re-target this PR without breaking everything, I'd appreciate it if you could do so. Otherwise, we can merge into `master` and we'll handle getting these changes into `main`.
- Looks like you've got a couple of merge conflicts, probably because we took so long to review this. Sorry about that!
- I have a few small requests for rephrasing or not doing some edits. Let me know if those work out okay for you!

File: alerts.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
-Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via the Kubecost UI or via Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
+Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via the Kubecost UI or Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
   
 
 As of v1.72.0, Kubecost supports four types of notifications:
 
  1. [Recurring update](#type-recurring-update) - sends an email and/or Slack alert with cluster spend across all or a set of namespaces, with costs broken down by namespace
 
- 2. [Budget](#type-budget) -- sends an email and/or Slack alert when spend crosses a defined threshold
+ 2. [Budget](#type-budget) -- sends an email and/or Slack alert when spending crosses a defined threshold
```
[the remark this answers is not in the document]
Comment (IvanCherepov) [reply alerts.md]: fixed

File: multi-cloud.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Setting Up Cloud Integrations
-This document outlines how to set up cloud integration for accounts on multiple cloud providers, or multiple acounts on the same cloud provider. Multi-Cloud is an enterprise feature. This configuration can be used independent of or in addition to other cloud integration configurations provided by kubecost. Once configured Kubecost will display cloud assets for all configured account and perform reconcilation for all [federated clusters](https://github.com/kubecost/docs/blob/master/long-term-storage.md) that have there respective accounts configured.
+This document outlines how to set up cloud integration for accounts on multiple cloud providers, or multiple accounts on the same cloud provider. Multi-Cloud is an enterprise feature. This configuration can be used independently of or in addition to other cloud integration configurations provided by Kubecost. Once configured Kubecost will display cloud assets for all configured accounts and perform reconciliations for all [federated clusters](https://github.com/kubecost/docs/blob/master/long-term-storage.md) that have their respective accounts configured.
```
[the remark this answers is not in the document]
Comment (IvanCherepov) [reply multi-cloud.md:2]: fixed

File: allocation-api.md [hunk 3]
```diff
@@ -42,15 +42,15 @@ Optional request parameters include the following:
...
+`filterFields` | Blacklist of fields to be filtered from the response. For example, appending `&filterFields=cpuused,cpureq,ramreq,ramused` will remove request and usage data.
 `namespace` | Filter results by namespace. For example, appending `&namespace=kubecost` only returns data for the `kubecost` namespace
 
 
 # Aggregated cost model API
 
 > NOTE: this API is actively being replaced by the [Kubecost Allocation API](https://github.com/kubecost/docs/blob/master/allocation.md). That is the recommended API for querying historical and run-rate cost allocation metrics.
 
-The aggregated cost model API retrieves data similar to the Kubecost Allocation frontend view (e.g. cost by namespace, label, deployment and more) and is available at the following endpoint:
+The aggregated cost model API retrieves data similar to the Kubecost Allocation frontend view (e.g. cost by namespace, label, deployment, and more) and is available at the following endpoint:
```
[the remark this answers is not in the document]
Comment (IvanCherepov) [reply allocation-api.md:53]: :)

**Comment by IvanCherepov** [comment]
@michaelmdresser You are welcome! Unfortunately, I don't know good git linter for PRs. Although I think I addressed all of your comments, rebase might take a lot of effort. Do you think it'll be a good idea to close this PR and start a new one from the correct branch?

**Review by michaelmdresser — APPROVED** [review]
This is perfect now @IvanCherepov, thank you so much for the contribution.

**Pull request merged**